### ISSUE TITLE
Terminal plugin 

### ISSUE THREAD

**Issue by swatrigant** [Atheos/Atheos#111]
Hello ! 

I really want to use the Terminal plugin, but can't make it work because it is still the "old" plugin, the Codiad one, and I'm having a hard time updating it. Do you think you could work on updating this particular plugin ? 

I fully understand if it is not a priority right now, or if you don't have time to work on that issue ! I think it is amazing how quickly you react to any issue, thank you !

**Comment by HLSiira** [comment]
Let me see what I can work out. Have you used the plugin before on Codiad?

If I recall correctly, it was a little clunky so I can probably get it working soon but it might not be much better than it was until I spend some rime cleaning it

-------- Original Message --------

**Comment by swatrigant** [comment]
Yes, it was a little it old-fashion on Codiad, but it will do (at least for me). I was thinking about making it work (but I can't, which is why I opened a new issue) and then later maybe making it look more modern !

Thanks again !

**Comment by HLSiira** [comment]
If I may ask, what are you getting hung up on?

**Comment by HLSiira** [comment]
Hey @swatrigant,

I got it updated and pushed to the master repo, but I highly suggest you pull from the development repo of Atheos in order to ensure that you're working with the latest source as that's what I run and made it work with Terminal.

I offer no promises that it will work perfectly since I just kind of did a quick copy-paste of the original codiad repo and all it's pull requests, merging with zero checks. I haven't tested everything yet but as far as I can tell, no bugs.

I also went a little too into the styling side of the terminal since I wanted to; hopefully you're into the style.

Let me know what you think or any bugs or errors you have.
